app/controllers/saml_sessions_controller.rb

```diff
@@ -15,4 +15,43 @@ def set_user_session_index
   def get_user_session_index
     current_user.session_index = session[:saml_session_index]
```

Comment:
Should this line use of the Devise.saml_session_index_key config? If it did, the session variable name would match the config instead of being hardcoded

Reply:
Same question for line 12.